pkg/agent/plugin/nodeattestor/aws/iid.go

```diff
@@ -36,8 +40,9 @@ func builtin(p *IIDAttestorPlugin) catalog.Plugin {
 
 // IIDAttestorConfig configures a IIDAttestorPlugin.
 type IIDAttestorConfig struct {
-	IdentityDocumentURL  string `hcl:"identity_document_url"`
-	IdentitySignatureURL string `hcl:"identity_signature_url"`
+	EC2MetadataEndpoint  string `hcl:"ec2_metadata_endpoint"`
+	IdentityDocumentURL  string `hcl:"identity_document_url"`  // Deprecated
```

Comment:
nit: prefix vars with `Deprecated` (but keep hcl tag as-is) as a visual indicator that new code shouldn't rely on them